go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/ONSdigital/dp-dataset-api
+
+go 1.13
+
+require (
+	github.com/ONSdigital/dp-api-clients-go v1.9.0
+	github.com/ONSdigital/dp-authorisation v0.0.0-20191018110224-d4543527d6cb
```

Comment:
Is there a release version we can use for this?

Reply:
Not yet and I think the library needs more work before we can create version 1